graphql/schema/request.go

```diff
@@ -32,10 +32,18 @@ type Request struct {
 	Query         string                 `json:"query"`
 	OperationName string                 `json:"operationName"`
 	Variables     map[string]interface{} `json:"variables"`
+	Extensions    struct {
```

Comment:
Leave a comment here, that we should add explicit types for all extensions that we support in dgraph